### ISSUE TITLE
SeisBench EQTransformer

### ISSUE THREAD

**Issue by chenglongduan** [seisbench/seisbench#294]
I was told that EQTransformer will generate different probability traces at different runs.
So, I do experiments using the EQTransformer functions in SeisBench. I make two identical input data by copy and paste (namely D1 and D2, D1=D2).
I run D1 to obtain EQT traces in stream (namely EQT1). Same thing, I run D2 to obtain EQT2.
After comparing the detailed values of P and S traces in EQT1 and EQT2, I find they are exactly the same.
Just wonder if I am correct about this?

The version of SeisBench that I use is 0.4.0. And below is partial of my code related to seisbench:
```
import seisbench.models as sbm
# parameters
trained_model = 'eqt'
trained_data = 'stead'
seisbench_sample_rate = 100.0

# load EQT
if trained_model=='eqt':
    seisbench_model = sbm.EQTransformer.from_pretrained(trained_data)

# resample stream data to 100 Hz, as required by seisbench
seis.resample(seisbench_sample_rate, window='hann')

# generate probability traces
seis_picks = seisbench_model.annotate(seis)
```

**Comment by yetinam** [comment]
Hi @chenglongduan ,

thanks for your question. Indeed, EQTransformer predictions in SeisBench are deterministic, i.e., for the same input you'll always get the same output. This is intended for to ensure reproducibility. The way you can get different outputs is called Bayesian dropout. For this, you keep the dropout layers active in evaluation instead of deactivating them. You then draw multiple samples to get a statistical distribution. This function is currently not implemented in SeisBench, but I'm sure you could hack it in. Essentially, you'd need to comment out the relevant `model.eval()` call in the backend of `annotate`.

As a side remark, you do not need to resample the data yourself. SeisBench will anyhow do so internally.
